clusterloader2/testing/load/deployment.yaml

```diff
@@ -34,6 +34,9 @@ spec:
             mountPath: /var/secret
       dnsPolicy: Default
       terminationGracePeriodSeconds: 1
+{{if not ( Chance (DefaultParam .CL2_LOAD_DEPLOYMENTS_AUTOMOUNT_SERVICE_ACCOUNT_TOKEN_CHANCE 1 ))}}
```

Comment:
Yes, it would have the same meaning, but this would mean that parameter would be renamed to something like CL2_LOAD_DEPLOYMENTS_AUTOMOUNT_SERVICE_ACCOUNT_TOKEN_CHANCE_TO_NOT_BE_MOUNTED, and I prefer to have a simpler API from client perspective.

Reply:
CL2_DEPLOYMENT_DISABLE_AUTOMOUNT_DEFAULT_SERVICE_ACCOUNT_CHANCE

this is actually more user-friendly, because it is enabled by default, to you want to specify percentage chance of changing default behavior

Reply:
I was aiming in preventing adding a negation to a flag, but I can get an argument about changin default state.

Updated to `CL2_*_DISABLE_AUTOMOUNT_DEFAULT_SERVICE_ACCOUNT_CHANCE`

Reply:
What we should target it not "prevent negation', but think what is default.
By default automounting is enabled. So you don't need any action to enabled it. The action is to disable it, and thus the var should be around disabling, not enabling it.